website/docs/commands/basics.md

```diff
@@ -181,6 +181,20 @@ echo 'println("Hello")' | scala-cli -
 # Hello
 ```
 
+## ScalaCLI version
+
+`scala-cli` can also run the other version of `scala-cli`, it may be helpful to test unreleased scala-cli functionality.
```

Comment:
```suggestion
`scala-cli` can also run another Scala CLI version, which can be helpful to test unreleased yet Scala CLI functionality.
```

Reply:
I would also warn users that this runs a JVM-based Scala CLI, not a native one, so that this might be a bit slower.